static/js/message_edit.js

```diff
@@ -274,6 +274,12 @@ function edit_message(row, raw_content) {
     var editability = get_editability(message, seconds_left_buffer);
     var is_editable = editability === message_edit.editability_types.TOPIC_ONLY ||
                        editability === message_edit.editability_types.FULL;
+    var max_file_upload_size = page_params.max_file_upload_size;
+    var display_attachfile_icon = false;
+
+    if (max_file_upload_size > 0) {
+        display_attachfile_icon = true;
```

Comment:
Let's call this variable `file_upload_enabled` -- that's a lot more readable :)

Reply:
Changed!